README.md

```diff
@@ -1,2 +1,88 @@
-# MenuMingle
-MenuMingle is the graduation project of team 6 from the GazaSkyGeeks Code Academy bootcamp.
+# Menu Mingle <img src='./img/chef.png' style='height:30px'>
+our app allows clients to track restaurant orders. It provides an intuitive interface for placing orders, tracking their status, and managing the restaurant's menu.
+## Table of Contents
+- [Introduction](#Menu)
+- [Features](#features)
+- [ UserStory ](#User)
+- [Database Schema](#Database)
+- [Technologies Used](#Technologies)
+- [Demo](#Demo)
+- [Figma Design](#Figma)
+- [Contributing](#contributing)
+- [Acknowledgements](#Acknowledgements)
+# Features 
+- User registration and authentication for the staff.
+- Menu management for the restaurant.
+- Order placement by customers.
+- Real-time order tracking.
+- Notification system for order updates.
+- Admin dashboard for managing orders and menus.
+- supports English and Arabic language.
+# User Story
```

Comment:
This should e the user journey.

User journey illustrated the journey of our client on the website.
E.x.: When the user goes to a restaurant to eat, the user will find a QR taped to each table, after scanning this QR, our website will be opened and the user can order from the menu, (and you complete this way).